lib/index.js

```diff
@@ -204,13 +207,9 @@ module.exports.render = function(options) {
   var importer = options.importer;
 
   if (importer) {
-    options.importer = function(file, prev, key) {
+    options.importer = function(file, prev, bridge) {
       function done(data) {
-        console.log(data); // ugly hack
```

Comment:
Yes here: https://github.com/sass/node-sass/pull/615

> If anyone could ever explain this!

I was also intrigued about it. Took me sometime to *devise* this random hack. :D

It could have fixed in this overhaul. I am still very interested to know about what was causing it. :)